.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          GIT_DOMAIN_OPTION=""
+          if [[ -n "${{ vars.GIT_DOMAIN }}" ]]; then
+            GIT_DOMAIN_OPTION=" --git.domain=${{ vars.GIT_DOMAIN }}"
+          fi
+          STATIC_ANALYSIS_TOOL_OPTION=""
+          if [[ -n "${{ vars.STATIC_ANALYSIS_TOOL }}" ]]; then
+            STATIC_ANALYSIS_TOOL_OPTION=" --static_analysis_tool=${{ vars.STATIC_ANALYSIS_TOOL }}"
+          fi
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }}${GIT_DOMAIN_OPTION}${STATIC_ANALYSIS_TOOL_OPTION}" >> $GITHUB_ENV
+      - name: Code Review Agent - Issue Comment
+        if: github.event_name == 'issue_comment'
+        uses: gitbito/codereviewagent@main
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DjangoProject/pull/5#issuecomment-2168349471>#d58ef4</a><span> - 06/14/2024, 04:18 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The GitHub Action is using the 'gitbito/codereviewagent@main' action. If the 'main' branch of the action repository is updated with breaking changes, it could potentially disrupt the workflow.
</div>
<div id="fix">
<b>Fix:</b> Pin the action to a specific version or commit SHA to ensure stability and prevent unexpected issues due to updates in the action repository.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -31,7 +31,7 @@
       if: github.event_name == 'issue_comment'
       uses: gitbito/codereviewagent@v1.0.0
       with:
         pr: ${{ github.event.issue.pull_request.html_url }}
         command: ${{ github.event.comment.body }}
         options: ${{ env.OPTIONS }}
```
</div>
</details>
</div>
<br>


</div>